File: Matlab/Statistics/statistics.m
clc;
clear all;
close all;

data = csvread("..\..\Data\PETR4.SA.csv", 2, 1);

adj = data(:,5);

figure
plot(adj)
grid on

mean(adj)
var(adj)
std(adj)

figure
autocorr(diff(adj))

File: R/Statistics/statistics.R
PETR <- read.csv(file="..\\Data\\PETR4.SA.csv", header=TRUE, sep=",")

adClose <- PETR$Adj.Close

#we want to plot the adjusted closing prices
plot(adClose)

#this is the auto-correlation function ... get rid of the NA values
acf(diff(adClose),na.action=na.omit)

#statistics: mean, variance and standard deviation
mean(adClose)
var(adClose)
sd(adClose)

